### PR TITLE
Updated code quality binary operator line change & updated bare except warning code in class_objectProcessor.py

### PR DESCRIPTION
**Pull request by kdcis**
Binary operator line change update
Replaced BMConfigParser().get with BMConfigParser().safeGet and removed exception handling

### CODE REVIEW

File: src/class_objectProcessor.py [hunk 1]
```diff
@@ -727,7 +727,7 @@ def processmsg(self, data):
                 try:
                     mailingListName = BMConfigParser().get(
                         toAddress, 'mailinglistname')
-                except:
+                except:  # noqa:E722
```
Comment (PeterSurda) [inline src/class_objectProcessor.py:730]: `safeGet`, default empty string

File: src/class_objectProcessor.py [hunk 2]
```diff
@@ -1017,7 +1017,7 @@ def processbroadcast(self, data):
             try:
                 apiNotifyPath = BMConfigParser().get(
                     'bitmessagesettings', 'apinotifypath')
-            except:
+            except:  # noqa:E722
```
Comment (PeterSurda) [inline src/class_objectProcessor.py:1020]: same as above, `safeGet`, default `None`

File: src/class_objectProcessor.py [hunk 3]
```diff
@@ -1017,7 +1017,7 @@ def processbroadcast(self, data):
...
                 apiNotifyPath = ''
             if apiNotifyPath != '':
```
Comment (PeterSurda) [inline src/class_objectProcessor.py:1022]: `if apiNotifyPath:`

**Review by PeterSurda — CHANGES_REQUESTED** [review]

File: src/class_objectProcessor.py [hunk 4]
```diff
@@ -715,7 +715,7 @@ def processmsg(self, data):
                 try:
                     apiNotifyPath = BMConfigParser().get(
                         'bitmessagesettings', 'apinotifypath')
-                except:
+                except:  # noqa:E722
```
[the remark this answers is not in the document]
Comment (kdcis) [reply src/class_objectProcessor.py]: Updated

File: src/class_objectProcessor.py [hunk 5]
```diff
@@ -715,7 +715,7 @@ def processmsg(self, data):
...
                     apiNotifyPath = ''
                 if apiNotifyPath != '':
```
[the remark this answers is not in the document]
Comment (kdcis) [reply src/class_objectProcessor.py]: Updated

File: src/class_objectProcessor.py (hunk 1 above)
Comment (kdcis) [reply src/class_objectProcessor.py]: Updated

File: src/class_objectProcessor.py (hunk 2 above)
Comment (kdcis) [reply src/class_objectProcessor.py]: Updated

File: src/class_objectProcessor.py (hunk 3 above)
Comment (kdcis) [reply src/class_objectProcessor.py]: Updated

**Review by PeterSurda — APPROVED** [review]

**Comment by kdcis** [comment]
Code quality changes done & merged in @g1itch 's  PR #1832

**Pull request closed without merging**